dotCMS/src/main/java/com/dotcms/rest/api/v1/maintenance/MaintenanceResource.java

```diff
@@ -101,29 +115,86 @@ public final Response downloadLogFile(@Context final HttpServletRequest request,
             @Context final HttpServletResponse response, @PathParam("fileName") final String fileName)
             throws IOException {
 
-        final InitDataObject initData =
-                new WebResource.InitBuilder(webResource)
-                        .requiredBackendUser(true)
-                        .requestAndResponse(request, response)
-                        .rejectWhenNoUser(true)
-                        .requiredPortlet("maintenance")
-                        .init();
-
-            String tailLogFolder = Config
-                    .getStringProperty("TAIL_LOG_LOG_FOLDER", "./dotsecure/logs/");
-            if (!tailLogFolder.endsWith(File.separator)) {
-                tailLogFolder = tailLogFolder + File.separator;
-            }
-
-            final File logFile = new File(FileUtil.getAbsolutlePath(tailLogFolder + fileName));
-            if(!logFile.exists()){
-                throw new DoesNotExistException("Requested LogFile: " + logFile.getCanonicalPath() + " does not exist.");
-            }
-
-            Logger.info(this.getClass(), "Requested logFile: " + logFile.getCanonicalPath());
-
-            response.setHeader( "Content-Disposition", "attachment; filename=" + fileName );
-            return Response.ok(logFile, MediaType.APPLICATION_OCTET_STREAM).build();
+        assertBackendUser(request, response);
+
+        String tailLogFolder = Config
+                .getStringProperty("TAIL_LOG_LOG_FOLDER", "./dotsecure/logs/");
+        if (!tailLogFolder.endsWith(File.separator)) {
+            tailLogFolder = tailLogFolder + File.separator;
+        }
+
+        final File logFile = new File(FileUtil.getAbsolutlePath(tailLogFolder + fileName));
+        if(!logFile.exists()){
+            throw new DoesNotExistException("Requested LogFile: " + logFile.getCanonicalPath() + " does not exist.");
+        }
+
+        Logger.info(this.getClass(), "Requested logFile: " + logFile.getCanonicalPath());
+
+        response.setHeader( "Content-Disposition", "attachment; filename=" + fileName );
+        return Response.ok(logFile, MediaType.APPLICATION_OCTET_STREAM).build();
+    }
+
+    /**
+     * This method attempts to send resolved DB dump file using an octet stream http response.
+     *
+     * @param request  http request
+     * @param response http response
+     * @return octet stream response with file contents
+     * @throws IOException
+     */
+    @Path("/_downloadDb")
+    @GET
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_OCTET_STREAM, MediaType.APPLICATION_JSON})
+    public final Response downloadDb(@Context final HttpServletRequest request,
+                                     @Context final HttpServletResponse response) throws IOException {
+        assertBackendUser(request, response);
```

Comment:
Downloading the db needs to be admin only - not just maintenance portlet access.

Reply:
Done.